docker/dash/docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+# Common Docker compose configuration file for DASH
+
+lb:
+  build: ./lb
+  ports:
+    - "80:80"
+    - "443:443"
+  links:
+    - frontend
+    - backend
+  environment:
+    - DEFAULT_HOST=dash.akvotest.org
+  volumes:
+    - /var/run/docker.sock:/tmp/docker.sock:ro
+    - /etc/nginx/certs:/etc/nginx/certs
+  restart: always
+
+frontend:
+  build: ./frontend
+  environment:
+    - VIRTUAL_HOST=dash.akvotest.org
+
+backend:
```

Comment:
we should expose to the `backend` a host volume, e.g.

    volumes:
      - /var/tmp/akvo:/tmp/akvo